lms/djangoapps/branding/api.py

```diff
@@ -221,20 +222,23 @@ def _build_support_form_url(full_path=False):
     return contact_us_page
 
 
-def _build_help_center_url(language=settings.LANGUAGE_CODE):
+def _build_help_center_url(language):
     """
     Return the help-center URL based on the
     language selected on the homepage.
     :param language: selected language
     :return: help-center URL
     """
     support_url = settings.SUPPORT_SITE_LINK
-    enabled_languages = {
-        'en': 'hc/en-us',
-        'es': 'hc/es-419'
-    }
-    if language in enabled_languages:
-        support_url += '/' + enabled_languages[language]
+    # Changing the site url only for the Edx.org
+    # not for OpenEdx.
+    if support_url and 'support.edx.org' in support_url:
+        enabled_languages = {
+            'en': 'hc/en-us',
+            'es-419': 'hc/es-419'
+        }
+        if language in enabled_languages:
+            support_url = urljoin(support_url, enabled_languages[language])
```

Comment:
Add an empty line in between return and this line.

Reply:
Empty line added before return.